README.md

```diff
@@ -88,16 +95,17 @@ models:
       +schema: my_new_schema_name # leave blank for just the target_schema
 ```
 
-## Step 5: Finish Setup
-Your dbt project is now setup to successfully run the dbt package models! You can now execute `dbt run` and `dbt test` to see the models materialize in your warehouse and execute the data integrity tests applied within the package.
+### Step 5: Finish Setup
+You have successfully set up your dbt project to run the dbt package models! You can now execute `dbt run` and `dbt test` to see the models materialize in your destination and execute the data integrity tests applied within the package.
+
+### (Optional) Step 6: Orchestrate your package models with Fivetran
+Fivetran offers the ability for you to orchestrate your dbt project within the Fivetran dashboard. Learn how to orchestrate dbt models in your dashboard in the [Transformations for dbt Core documentation](https://fivetran.com/docs/transformations/dbt).
 
-## (Optional) Step 6: Orchestrate your package models with Fivetran
-Fivetran offers the ability for you to orchestrate your dbt project through the [Fivetran Transformations for dbt Core](https://fivetran.com/docs/transformations/dbt) product. Refer to the linked docs for more information on how to setup your project for orchestration through Fivetran. 
+## Package Dependency Matrix
```

Comment:
All of the Level 2 (`##`) headers before this section are questions. I think we should do this throughout for consistency and to maintain a conversational tone. What about "Does this package have dependencies?"

Reply:
Yeah I really like that idea! Keep the conversational tone is a great idea.